units/host_allocator.go

```diff
@@ -19,10 +19,13 @@ import (
 	"github.com/mongodb/grip/message"
 	"github.com/mongodb/grip/sometimes"
 	"github.com/pkg/errors"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
 )
 
 const (
-	hostAllocatorJobName = "host-allocator"
+	hostAllocatorJobName         = "host-allocator"
+	hostAllocatorAttributePrefix = "evergreen.host_allocator"
```

Comment:
I like this change a lot. If we had to rename these, grepping for "evergreen" would be a 😨